src/components/Form/Form.tsx

```diff
@@ -27,27 +25,25 @@ const Form = ({ submit }: FormProps) => {
 
   return (
     <FormProvider {...methods}>
-      <form className="form" onSubmit={onSubmit}>
-        <div className="form__contents">
-          <div className="textarea-input-container">
+      <form
+        className="bg-formBackground p-[28px] pt-7 pb-10 rounded-[40px] max-w-[467px] w-full"
+        onSubmit={onSubmit}
+      >
+        <div className="flex flex-col justify-center items-center">
+          <div className="w-full mb-4">
             <TextAreaInput
               name="description"
               placeholder="Give us some details about your event"
             />
           </div>
-          <CheckboxInput
-            name="agreeToTerms"
-            label="I agree to receive automated calls or text messages from Expert Event Solutions LLC in response to my inquiry. Msg & data rates may apply."
-            error={{
-              message: methods.formState.errors.agreeToTerms?.message ?? "",
-            }}
-            //formState property contains the errors object, which contains validation errors for each form field.
-            //CheckboxInput component, the error prop is used to display an error message if the agreeToTerms checkbox input is not checked.
-            // message property that contains the error message.
-            //methods.formState.errors.agreeToTerms?.message expression checks if the agreeToTerms field has a validation error. If it does, the error message is displayed.
-            //?? "" operator is used to provide a default value of an empty string if the message property is undefined
-          />
-          <button type="submit">Submit</button>
+
+          <button type="submit" className="mt-4">
```

Comment:
The submit button is a different task (EES 11). 
Maybe share the code with Marcell before you delete this.